lib/jellyfish/room_service.ex

```diff
@@ -176,7 +178,7 @@ defmodule Jellyfish.RoomService do
   def handle_info({:DOWN, _ref, :process, pid, reason}, state) do
     {room_id, state} = pop_in(state, [:rooms, pid])
 
-    Logger.warning("Process #{room_id} is down with reason: #{reason}")
+    Logger.warning("Process #{room_id} is down with reason: #{inspect(reason)}")
```

Comment:
If reason is an atom or string we dont need inspect here

Reply:
Yes I added this because once I receive something different, if I remember it was a tuple and this leads to error but I can remove it.